### DISCUSSION TITLE
Strange messages when using suggested docker compose setup

### DISCUSSION THREAD

**Discussion by lramos15** [directus/directus#14839] [Archive]
👋 ,

I'm trying to use the suggested docker compose configuration found on https://docs.directus.io/self-hosted/installation/docker/ and it starts up and appears to work fine, but the logs are a bit weird. I've included a picture below and would love to know if these are expected or how to fix them

<img width="644" alt="image" src="https://user-images.githubusercontent.com/4544166/182728629-2a716917-27a8-4fe5-af16-9292d08875af.png">

1. `PUBLIC_URL should be a full URL`: this appears expected because I did not set it
2. ` Upload directory (/directus/uploads) is not read/writeable!`: Not sure how to fix this. As I mount the uploads volume into the container as demonstrated in the docker example
3. There's a lot of lines with `return` and `return in set Commander`: These seem to really fill up the logs and would prefer to hide them if possible. `return in set Commander` appears related to redis, but not sure about just the blank return.

Thanks for the help and let me know if you need any further info!

**Comment by rijkvanzanten** [comment]
Known issue in a third party dependency, see https://github.com/directus/directus/issues/14812 for more information :)

**Reply by lramos15** [reply]
Thanks! Do you know how I fix the second issue? Docker has sudo permission so I assume that the /directus/uploads directory would too so I'm unsure how to fix this

**Reply by rijkvanzanten** [reply]
I don't know. I'm assuming there's a problem with the volume? Not sure!
